File: backend/src/main/java/at/home/freezerlist/FreezerlistApplication.java
package at.home.freezerlist;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;

@SpringBootApplication
@EnableJpaRepositories
public class FreezerlistApplication {

    private static final Logger log = LoggerFactory.getLogger(FreezerlistApplication.class);

    public static void main(String[] args) {
        SpringApplication.run(FreezerlistApplication.class, args);
    }

}

File: src/main/kotlin/at/home/freezerlist/FreezerlistApplication.kt
package at.home.freezerlist

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.web.bind.annotation.RequestMapping

@SpringBootApplication
class FreezerlistApplication

fun main(args: Array<String>) {
	runApplication<FreezerlistApplication>(*args)
}
